src/Umbraco.Web.UI.Client/src/views/common/infiniteeditors/propertysettings/propertysettings.controller.js

```diff
@@ -199,6 +199,7 @@
 
         function submitOnEnter(event) {
             if(event && event.keyCode === 13) {
+                event.preventDefault();
```

Comment:
Hi @matthewcare , thanks for tackling this issue 👍
I did test this locally and I still get a form submit when hitting enter in the name field. Looking at the code here, I was wondering if we should not remove the "submit"? Because I guess that preventing default and then calling submit kind of cancels the effect? Or is there something wrong with my browser 😅 ?

Cheers!